collection-scripts/gather_monitoring

```diff
@@ -111,6 +112,17 @@ monitoring_gather(){
 
   # TODO: Add out of the box metrics.
   # metrics_gather --min-time=${promtool_min_time} --match=xxx || true
+  echo "INFO: Gathering metrics from Prometheus replicas"
+  METRIC_MATCHES=(
```

Comment:
As I commented in here https://github.com/openshift/must-gather/pull/490#pullrequestreview-2912067414, we can't accept any new log that increases the storage size and elapsed time. So that we need data to compare against the main branch. How much data will this add on our average size?. Will this be significant if there are 1K nodes?.